src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.tsx

```diff
@@ -70,19 +70,20 @@ export const BlastSubmissionHeader = (props: Props) => {
     submission.submittedData.parameters.program.toUpperCase();
   const submissionId = submission.id;
   const submissionTime = getFormattedDateTime(new Date(submission.submittedAt));
+  const { jobName } = submission.submittedData.parameters;
 
   const editSubmission = () => {
     const { sequences, species, preset, parameters } = submission.submittedData;
     const parsedSequences = sequences.flatMap((sequence) =>
       parseBlastInput(sequence.value)
     );
-    const { title, program, stype, ...otherParameters } = parameters;
+    const { jobName, program, stype, ...otherParameters } = parameters;
 
     const payload = {
       sequences: parsedSequences,
       selectedSpecies: species,
       settings: {
-        jobName: title,
+        jobName: jobName,
```

Comment:
Can just be `jobName` rather than writing this as a key-value pair

Reply:
:heavy_check_mark: